spack-ext/lib/jcsda-emc/spack-stack/stack/templates/mpi.lua

```diff
@@ -32,11 +32,19 @@ setenv("MPI_F77", "@MPIF77@")
 setenv("MPI_F90", "@MPIF90@")
 
 -- intel specific mpi wrapper environment variables
-setenv("I_MPI_CC",  os.getenv("CC"))
-setenv("I_MPI_CXX", os.getenv("CXX"))
-setenv("I_MPI_F77", os.getenv("F77"))
-setenv("I_MPI_F90", os.getenv("FC"))
-setenv("I_MPI_FC",  os.getenv("FC"))
+local i_mpi_cc = os.getenv("CC")
+local i_mpi_cxx = os.getenv("CXX")
+local i_mpi_f77 = os.getenv("F77") or ""
```

Comment:
why only for f77 and not for all?

Reply:
Not sure if the F77 is defined in the first place; it may not be needed and possibly is left for the legacy reasons.
i_mpi_f77 is therefore not used in the conditional statement further below in line 41.